.github/ISSUE_TEMPLATE/1-bug.yaml

```
@@ -0,0 +1,59 @@
+name: Bug Report
+description: Open an issue for a problem you've identified with `fuels`.
+labels: ["triage"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for helping to improve our Typescript SDK!
+
+        Before submitting an issue, please ensure you are using the latest version of the SDK, as well as `forc` and `fuel-core`. Additionally, search through
+        the [documentation](https://docs.fuel.network/docs/fuels-ts/) or on our [forum](https://forum.fuel.network/c/typescript-sdk) as it's possible your problem
+        has already been solved there.
```

```suggestion
        Before submitting an issue, please ensure you are using the latest version of the SDK, as well as `forc` and `fuel-core`. Additionally, search through the [documentation](https://docs.fuel.network/docs/fuels-ts/) or on our [forum](https://forum.fuel.network/c/typescript-sdk) as it's possible your problem has already been solved there.
```

Some odd line breaks in the rendered version here